static/app/views/replays/replaySearchBar.tsx

```diff
@@ -53,6 +69,10 @@ function ReplaySearchBar(props: Props) {
   const {organization, pageFilters} = props;
   const api = useApi();
   const projectIdStrings = pageFilters.projects?.map(String);
+  const tags = useTags();
+  useEffect(() => {
+    loadOrganizationTags(api, organization.slug, pageFilters);
+  }, [api, organization.slug, pageFilters]);
```

Comment:
We could choose to call this higher up in the component tree to make sure it's loaded sooner, but I've chosen to colocate it with the search bar that requires it. Would make sense to move it if other components require loading tags as well.